### DISCUSSION TITLE
Drive not detected after flashing micropython

### DISCUSSION THREAD

**Discussion by htc-htc** [micropython/micropython#13740] [RP2040 / Pico] [answered]
Hi, I'm just got a YD-RP2040 board and plugged it in for the first time. The usb drive was soon detected and I flashed the board with the micropython downloaded from the link provided on the index.htm file on the drive (https://www.raspberrypi.com/documentation/microcontrollers/micropython.html). The version of the micropython I flashed was RPI_PICO-20240222-v1.22.2.uf2, however, I was never able to see the usb drive being detected again.
Things I've tried:
1. I've tried to use the reset method where I hold the boot button and and reset button but had no luck.
2. I have also tried to short the ground pin with the run pin but still didn't work.

I've seen a lot of people talking about using the nuke uf2 file but since I can't see the drive detected, I can't flash the file to use it.

Is there any other way to de-brick my board?

ps. I've tried these on a linux and a macos system.

Thanks!

**Comment by robert-hh** [comment, marked as answer]
The RPI drive is only used for uploading the firmware. During use there is no drive. You can upload python code to the device using e.g. mpremote.py. If the firmware update worked, you will get a REPL  prompt using  a serial terminal. mpremote suits too as serial terminal.

mpremote: https://docs.micropython.org/en/latest/reference/mpremote.html

**Reply by htc-htc** [reply]
Thanks very much for answering my stupid question, the mpremote program has been very helpful! Thanks again!
